lib/esbonio/CHANGES.rst

```diff
@@ -1,3 +1,11 @@
+Under development
+-----------------
+
+- Add ``textDocument/hover`` support. (`#311 <https://github.com/swyddfa/esbonio/issues/311>`_)
```

Comment:
No need to manually edit the `CHANGES.rst` file, this will be done by CI during release. 
Instead could you drop these in the `lib/esbonio/changes` folder in a file named something like `311.feature.rst`?